collections/forms/i18n/en.pot

```diff
@@ -99,3 +96,11 @@ msgstr "Please provide a string"
 
 msgid "Please provide a valid url"
 msgstr "Please provide a valid url"
+
+msgctxt " @ or #"
+msgid ""
+"Please provide a username between 4 and 255 characters long and possibly "
+"separated by . "
+msgstr ""
+"Please provide a username between 4 and 255 characters long and possibly "
+"separated by . _ @ or #"
```

Comment:
Looks like there's something funny going on with this string

Reply:
I don't know why this is in here, will remove from this PR

Reply:
Probably needs to be extracted somewhere else - I think the characters here are problematic

Reply:
Reverted for now